bin/configure-fastly.js

```diff
@@ -18,7 +19,15 @@ const extraAppRoutes = [
     '/[^/]*.html$'
 ];
 
-const routes = routeJson.map(
+const routeJsonPreProcessed = routeJson.map(
+    (route) => {
+        if (route.redirect) {
+            route.redirect.replace('RADISH_URL',RADISH_URL);
```

Comment:
```suggestion
    route => {
        if (route.redirect) {
            route.redirect.replace('RADISH_URL', RADISH_URL);
```

Reply:
THX, I went through the process of getting it running locally, and after getting through the cairo related build trouble on node-gyp, now can run linting locally :)